[OB3] Mention the security vulnerability specified in CVE-2019-13990 for quartz.properties (#543)
wso2/docs-open-banking: en/docs/install-and-setup/configuring-identity-server-for-ob.md

    
    ??? info "Click here to see the configurations for periodical consent expiration in a clustered setup."
       

>         - :exclamation: It's important to note that, `quartz.properties` is not recommended to use for initializing custom job data from an XML file, As there's a security vulnerability specified in CVE-2019-13990.

Reviewer: ```suggestion
        - :exclamation: Due to a security vulnerability specified in CVE-2019-13990, `quartz.properties` is not recommended for initializing custom job data from an XML file.
```
